eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * File containing ReindexCommand class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\Core\Persistence\Database\DatabaseHandler;
+use eZ\Publish\SPI\Persistence\Content;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\ProgressHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use eZ\Publish\SPI\Persistence\Content\ContentInfo;
+use eZ\Publish\SPI\Search\Handler as SearchHandlerInterface;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use RuntimeException;
+use PDO;
+
+class ReindexCommand extends ContainerAwareCommand
```

Comment:
but some class doc is always good, more important then the file doc that we kind of half way decided we will start to skip _(ref `File containing ReindexCommand class.`)_.